app/views/admin/audit_logs/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Audit Logs</h1>
+
+<%= render 'search_form' %>
```

Comment:
No need for a partial here. We aren't going to reuse it, are we?

Reply:
I wanted to separate out the form because it's likely this will change soon. I'll remove this for now.